diff -urN meld-3.20.2/bin/meld meld-3.20.2n/bin/meld
--- meld-3.20.2/bin/meld	2020-02-16 06:32:20.000000000 +0800
+++ meld-3.20.2n/bin/meld	2020-02-26 20:41:23.040177235 +0800
@@ -187,7 +187,7 @@
 
     gtk_requirement = (3, 20)
     glib_requirement = (2, 48)
-    gtksourceview_requirement = (3, 20, 0)
+    gtksourceview_requirement = (4, 2, 0)
 
     def missing_reqs(mod, ver, exc=None):
         if isinstance(exc, ImportError):
@@ -216,15 +216,6 @@
     except (ImportError, AssertionError, ValueError) as e:
         missing_reqs("GLib", glib_requirement, e)
 
-    try:
-        gi.require_version('GtkSource', '3.0')
-        from gi.repository import GtkSource
-        # TODO: There is no way to get at GtkSourceView's actual version
-        assert hasattr(GtkSource, 'SearchSettings')
-        assert hasattr(GtkSource, 'Tag')
-    except (ImportError, AssertionError, ValueError) as e:
-        missing_reqs("GtkSourceView 3", gtksourceview_requirement, e)
-
 
 def setup_resources():
     from gi.repository import GLib
